tensorboard/plugins/text/summary_v2.py

```diff
@@ -26,6 +26,40 @@
 def text(name, data, step=None, description=None):
     """Write a text summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`, `tf.summary.image`.
```

Comment:
Maybe personal taste, but I think that seeing '.scalar, .SummaryWriter, .image' is a bit redundant with the left sidepane for navigation in the devsite, especially if all tf.summary.* pages will mention this.

wdyt about something more generic along the lines of: "See the rest of the `tf.summary.*` methods for ways to write other data types"?

imo, this "see also" might make more sense at the bottom, near the mention of the demo notebook.
wdyt?

Reply:
I put these here to be consistent with the ui pattern for TensorFlow's https://www.tensorflow.org/api_docs/python/tf/concat

Note that the See Also section is handled specially by the docsite software.

Reply:
Gotcha, after seeing tf.concat's docs, this makes sense.  I'll also add similar "See also"s for .image, .histogram!